server/src-exec/Main.hs

```diff
@@ -26,7 +27,16 @@ runApp (HGEOptionsG rci hgeCmd) =
   withVersion $$(getVersionFromEnvironment) case hgeCmd of
     HCServe serveOptions -> do
       (initCtx, initTime) <- initialiseCtx hgeCmd rci
-      runHGEServer serveOptions initCtx initTime
+      latch <- liftIO newShutdownLatch
+      -- Catches the SIGTERM signal and initiates a graceful shutdown. Graceful shutdown for regular HTTP
+      -- requests is already implemented in Warp, and is triggered by invoking the 'closeSocket' callback.
+      -- We only catch the SIGTERM signal once, that is, if the user hits CTRL-C once again, we terminate
+      -- the process immediately.
```

Comment:
This is just a style comment, but [our style guide now states comments should be wrapped to 80 characters](https://github.com/hasura/graphql-engine/blob/1211c23a2fa0cefa81c1258e7f8071c7da5c6bbf/server/STYLE.md#line-length).